src/flypad-backend/src/Aircraft/AircraftProcedures.h

```diff
@@ -29,272 +28,264 @@ struct ProcedureStep {
 // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
 // @formatter:off
-static vector<ProcedureStep*>* POWERED_CONFIG_ON = new vector<ProcedureStep*>{
+static const std::vector<ProcedureStep> POWERED_CONFIG_ON = {
```

Comment:
An even better solution would be to use `std::array` instead of `std::vector` also because you already added the include but never used it. Also it would be better to move the declaration inside of the `AircraftProcedures` class to make it "private"